### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by monkrus**
Add an optional  "Advertise here" table .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
 
+
+  
+<body>
+<table>
+  <tr>
+    <th> PROMOTE YOURSELF AND PUT YOUR ADD HERE !
+  </tr>
+  <tr>
+    <th> SEND US A MESSAGE FOR DETAILS!
```
Comment (Ebazhanov) [inline README.md:15]: Again a very good idea @monkrus ! but would recommend you take these responsibilities on your own, so I put your email there!

```suggestion
    <th>SEND US AN <a href = "mailto: sergeisqa@gmail.com">EMAIL </a>FOR DETAILS!
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Sergey FYI I left some comment, please check it out!

**Review by Ebazhanov — APPROVED** [review]
Thanks go ahead! Approved!!!

**Review by monkrus — COMMENTED** [review]
Great idea!

**Pull request merged**